api-reference/10 UI Widgets/GridBase/3 Methods/getVisibleColumnIndex(id).md

```diff
@@ -3,13 +3,13 @@ id: GridBase.getVisibleColumnIndex(id)
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Gets the index of a visible column.
 
 ##### return: Number
-<!-- Description goes here -->
+The column's index.
 
 ##### param(id): Number | String
-<!-- Description goes here -->
+The ID of a visible column whose index should be returned.  
```

Comment:
The ID can be many things (see [columnOption(id)](https://js.devexpress.com/Documentation/ApiReference/UI_Widgets/dxDataGrid/Methods/#columnOptionid)). We can describe this parameter like [here](https://js.devexpress.com/Documentation/ApiReference/UI_Widgets/dxDataGrid/Methods/#columnOptionid_optionName).